docs/deployment-itch-io.md

```diff
@@ -0,0 +1,45 @@
+# Deploying to itch.io
+
+Before deploying your game to [itch.io](https://itch.io/) you will need to have your exported game files. You will also need an account and some screenshots of your project. If you are unsure how to export your game, please see the [Building Your Game](building-your-game.md) section.
+
+Here's what we will do:
+
+[[toc]]
+
+## Creating a project on itch.io
+
+Once you have logged in to your account, select the [Dashboard](https://itch.io/dashboard) tab and check that you are in the `Projects` tab in the "Creator Dashboard".
+
+![](./images/buildingAndDeployment/Deployment/itch.io/deployment-itch.io-dashboard.png)
+
+Press the **`Create new project`** button.
+
+::: tip
+There are many options in when you deploy your game to [itch.io](https://itch.io/), we will only cover the basics needed to get your game deployed. Feel free to experiment in this section or consult the official [itch.io docs](https://itch.io/docs/itch/) for more detailed guides.
+:::
+
+Lets fill in the following fields:
+
+- **Title** Enter your games title here.
+- **Project URL** This should automatically be generated for you, it is the URL where your project will be hosted.
+- **Kind of project** For this example we will choose HTML as we will be uploading a web version of our game.
+- **Upload Cover Image** Lets add a cover image so that players can see our game.
```

Comment:
Maybe using semicolons would be better? Like "**Title:** Enter your yadayada"
Reading them without any punctuation feels wrong.